web/src/components/Incidents/processIncidents.ts

```diff
@@ -62,17 +57,15 @@ export function processIncidents(data: Incident[]): ProcessedIncident[] {
 
     const firstDate = incident.values[0][0];
     const lastDate = incident.values[incident.values.length - 1][0];
-    const dayDifference = (lastDate - firstDate) / (1000 * 60 * 60 * 24);
-
-    const hasMatchingDayAndHour = incident.values.some((v) => {
-      const valueDate = new Date(v[0] * 1000);
-      return (
-        valueDate.getUTCDate() === currentDay &&
-        valueDate.getUTCMonth() === currentMonth &&
-        valueDate.getUTCFullYear() === currentYear &&
-        valueDate.getUTCHours() === currentHour
-      );
-    });
+    const currentDate = new Date(); // Current timestamp in milliseconds
+
+    // Calculate the time difference in milliseconds
+    const timeDifference = currentDate.valueOf() - lastDate * 1000;
+    const inactive = timeDifference < 10 * 60 * 1000;
```

Comment:
So the incident is considered inactive in 10 * 60 * 1000 = 600,000 ms = 10 minutes? Is that correct?

Reply:
Yes